tests/Unit/RoleTest.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Tests\Unit;
+
+use Tests\TestCase;
+use App\Alexa\Models\Role;
+use App\Alexa\Models\User;
+
+class RoleTest extends TestCase
+{
+    public function testRoleIsAdmin()
+    {
+        $role = factory(Role::class)->states('admin')->create();
```

Comment:
Let's make it differently.

Remove your factories (including factory file) and update your tests to the following:

```php

$role = new Role();
$role->type = Role::ROLE_ADMINISTRATOR;

... (asserts)
```

Reply:
👍